task/send-to-webhook-discord/0.1/send-to-webhook-discord.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: send-to-webhook-discord
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Messaging
+    tekton.dev/tags: messaging
+    tekton.dev/displayName: "Send a message to Discord channel"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
+spec:
+  description: >-
+    These tasks post a message to a Discord channel.
+
+    This task uses Webhooks integration of Discord channel to send a message.
+
+  params:
+  - name: webhook-secret
+    type: string
+    description: secret name of Discord webhook URL
+  - name: message
+    type: string
+    description: plain text message
+    default: 'Hello World from Tekton Bot'
+  - name: bot-name
+    type: string
+    description: plain text message
+    default: 'Tekton Bot'
+  - name: avatar-url
+    type: string
+    description: url avatar image
+    default: 'https://raw.githubusercontent.com/tektoncd/triggers/main/tekton-triggers.png'
```

Comment:
Non blocking: Shouldn't the default image be of Tekton pipelines and not of Tekton triggers?

Reply:
@vinamra28  yeah. the problem is i can't find any public static url for Tekton pipelines, thats why i use other Tekton project,
perhaps, can you give me URL image for Tekton pipelines?

Reply:
@ec2ainun, see if https://assets.openshift.com/hubfs/images/logos/Logo-tekton.svg works for you

Reply:
@vinamra28 is there any jpg or png format? i just test it out, but it does not work with svg, the image is not rendering.

Reply:
cc @vdemeester tekton-pipelines logs jpb/png
🐱

Reply:
@vinamra28 @nikhil-thomas @vdemeester should we upload new image for Tekton pipelines somewhere in this github repository? so we can reference it in the future in case someone need it, just like Tekton trigger do in https://raw.githubusercontent.com/tektoncd/triggers/main/tekton-triggers.png

Reply:
you can just use the github logo ? (Every org/user has a direct link) : 

https://github.com/tektoncd.png

Reply:
@chmouel thank you

Reply:
```suggestion
    default: 'https://github.com/tektoncd.png'
```